crypto/sr25519/sr25519test/keys.go

```diff
@@ -0,0 +1,43 @@
+package sr25519test
+
+import (
+	"log"
+
+	"github.com/mailchain/mailchain/crypto"
+	"github.com/mailchain/mailchain/crypto/sr25519"
```

Comment:
`"github.com/mailchian/mailchain/crypto/sr25519"` should be removed
Looking in the files for `sr25519.XXXXX` and remove `sr25519.` it is part of the same package

Reply:
done